src/Core/Enums/ButtonAppearance.cs

```diff
@@ -10,20 +10,15 @@ namespace Microsoft.FluentUI.AspNetCore.Components;
 public enum ButtonAppearance
 {
     /// <summary>
-    /// The button appears with the default style
+    /// Removes background styling.
     /// </summary>
-    Default,
```

Comment:
Why did you remove the `Default` enum and replaced it by `Outline` (like describe in the Migration/FluentButton.md file)?
This is not the same result.
![peek_1](https://github.com/user-attachments/assets/cfc3e91a-1894-4c6c-a040-dbaeb968052c)

You can update the **FluentButton.Appearance** parameter to be nullable, but not to remove this enum value.